src/library/Uno.Material/Styles/Controls/v2/ToggleButton.xaml

```diff
@@ -605,7 +605,7 @@
 						<!-- Alternate Content -->
 						<ContentPresenter x:Name="AlternateContentPresenter"
 										  AutomationProperties.AccessibilityView="Raw"
-										  Content="{Binding Path=(um:ControlExtensions.AlternateContent), RelativeSource={RelativeSource TemplatedParent}}"
+										  Content="{Binding Path=(um:ControlExtensions.AlternateContent), RelativeSource={RelativeSource TemplatedParent}, FallbackValue=' ', TargetNullValue=' '}"
```

Comment:
This would be the ToggleButton would become blank when toggled if you do not set an AlternateContent. Does it make more sense if we just simply do not switch Visibility between both ContentPresenters unless AlternateContent is actually set to something? 

Thoughts @Xiaoy312 / @agneszitte ?

Reply:
@kazo0 / @Xiaoy312 / @lesterbotello 
I think we can do like for the Icon and use `x:Null` in this case

Example here :
https://github.com/unoplatform/Uno.Themes/blob/75e56ee3d169f84781cb3bfdce03c2917f6d2832/src/library/Uno.Material/Styles/Controls/v2/PasswordBox.xaml#L208

Reply:
@lesterbotello Sorry I missed that it was already there for the `x:Null`

So we should have something like that for the Visibility
Visibility="{Binding Path=(um:ControlExtensions.AlternateContent), RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource MaterialNullToCollapsedConverter}, FallbackValue=Collapsed, TargetNullValue=Collapsed}" />

Reply:
Thanks @agneszitte, this also gets rid of the crash. Only thing is that this results in the same behavior mentioned by Steve (toggle button disappears if AlternateContent is not set), which I guess is not what we're looking for? (we actually want to skip changing `Visibility` entirely if is not set for `AlternateContentPresenter.Content` is not set.

Reply:
can i ask for a gif of current uwp behavior with this pr, and a gif from any of ios/droid/skia/wasm for comparison?

Reply:
@Xiaoy312 you most certainly can :)

Here's a GIF of the current behavior on Android (it's the same in the other platforms) and another one from Windows using the sample app attached to the issue. You may notice the Windows GIF stops abruptly after clicking on the first button, this is because the app crashes.


https://github.com/unoplatform/Uno.Themes/assets/30215699/53024615-cc0b-40fe-a3d5-cd56be3816c9

https://github.com/unoplatform/Uno.Themes/assets/30215699/84fd167e-fb1e-48f5-9341-03b31ea3070e

Reply:
In contrast, here's two more GIFs with the new behavior in the current PR (this is using Uno Gallery's own sample app)


https://github.com/unoplatform/Uno.Themes/assets/30215699/2cdce519-68d5-4f75-9324-f83109c18bf2



https://github.com/unoplatform/Uno.Themes/assets/30215699/55ea6b7b-8eb9-4ec4-8626-1641fcb77a80

Reply:
i dont see much value in throwing "is value null" check there, the fallback/target null is good enough
that said, we should be using opacity for toggle when possible, to avoid unnecessary measure pass (for performance)

Reply:
Updated. Also, we still need the Fallback/target null for the `Content` property of `AlternateContentPresenter` to prevent the crash.